### DISCUSSION TITLE
Welcome to Homebox Discussions!

### DISCUSSION THREAD

**Discussion by tankerkiller125** [sysadminsmedia/homebox#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NedPool** [comment]
Is this the new project for Homebox? I only saw the archive post from hay-kot today...  really hope the project isn't dead as it has real potential.

**Reply by tankerkiller125** [reply]
We are working to keep it going yes, we made a full "announcement" of sorts over on our shared blog: https://sysadminsjournal.com/were-continuing-homebox-development/

It should be noted that we did reach out to @hay-kot and he noted that he wouldn't be endorsing/blessing any forks or revitalization efforts until things shake out. Which we agree is probably the best course of action given the number of other projects out there that have been hijacked in the past and turned into malware or they themselves shutdown and archived (paperless went through at least 2 iterations of this before paperless-ngx came about as the current one).

We want to be clear, we're dedicated to keeping this thing alive, but it's going to take some time for us to learn the codebase, review existing issues on the original repo, go through the original repo pull requests, etc. but we'll work as fast as we reasonably can (given we also both have full time jobs).

But community contributions in the form of bug reports, feature requests, PRs, etc. are very welcome. We strongly believe that open source is strongest when it has a strong community that feels (and is) involved.

**Comment by jeranon** [comment]
This popped up on my unraid server as a docker container I could install. And I'm excited. I've been using a very unwieldly spreadsheet that I don't let my wife touch, because I'm paranoid she will break it. 

I have a few questions:
- I had to register to create a new account. Was there a default admin account that can be used for server settings that I missed?
- If I use a cloudflare tunnel to connect to this remotely (that would be amazing for adding contents of storage lockers), is there anything preventing anyone with the URL from registering and creating their own account? I guess what I'm asking is if it is possible to close registrations so only I can use it (or people I invite, too)

Again, Super excited about this project! Cheers!

**Reply by tankerkiller125** [reply]
The documentation under the quickstart section contains a list of environment variables you can take a look at. https://homebox.sysadminsmedia.com/quick-start.html

Notably, there is one called `HBOX_OPTIONS_ALLOW_REGISTRATION` that you can set to false to disable registration once your account is setup.

There are currently no server level admin settings or anything like that, they are all controlled via environment variables/config options.

**Reply by jeranon** [reply]
I did miss that, I've never edited a docker container after the fact, before. Thank you for the prompt reply!

**Comment by nickrout** [comment]
Hi, new user here. Would love to help. Not a coder, but can write docs etc. Pleased to see this happen. I set up homebox once before and then got distracted. I want to document my art collection, and hoping homebox may be the answer.

**Comment by Viper00o** [comment]
Hello All, I am a retired EE living in Kingman AZ.

### Education
- USAFA Class of 1979 LCWB (last all male class)
- BSEE Bioengeering Purdue 1981
- MSEE Microwave semiconductor device physics Purdue 1983

### Work Experience

I spent the 1st half of my career as a Microwave Design Engineer. Before WiFi I designed the 1st 2.4GHz spread spectrum radios for Symbol Technologies for their bar code scanners.

For the next 10 years I was a regulatory engineer for certification of WiFi radios for Symbol. 

I did a brief stint with Apple for regulatory approval of the 1st gen iPad and finished my career as a hardware test engineer at TCI an SPX corp.

All of my test software is written in NI LabView a graphical programming language that I use to control GPIB instruments in custom test stations. We used Microsoft TFS for code control.

### Retirement
I built my dream workshop for woodworking and am using HomeBox to track all of my hardware and tools and items in storage boxes.
